vuepress/docs/v6.3.2/tutorials/customize-the-platform/pda-tutorial.md

```diff
@@ -1,344 +1,233 @@
-implementation.
-
-### PAM Implementation
+6. Access the PDA plugn via one of the following options:
 
-The PAM implementation of forms depends on the presence of a form
-definition being present on the PAM task. The Entando PAM engine
-implementation transforms from the PAM format to the JSON schema to
-render the form. And it transforms the API format back to the PAM format
-based on the form definition in PAM. There are some limitations on form
-customization due to the format required to return data to PAM. See the
-forms section in the technical documentation for more information.
+**Option 1** If you'd like to make the PDA plugin your default home page, go to `App Builder → Pages → Settings`. In
+   the dropdown for Home Page, select `Home / xxxx` and click `Save`.
 
-## Attachments
+You can now navigate to your plugin's home page using the home icon in the upper right of the App Builder.
 
-The attachments MFE gives the user to view documents attached to a
-task/case/process and to add new documents to a task/case/process.
 
-### PAM Implementation
+**Option 2** Alternatively, you can view the PDA plugn home page by going to `Pages → Management`, finding `xxxx` in the page tree, and clicking `View Published Page` from its actions.
 
-The PAM implementation posts the documents to the PAM endpoints for
-storage. Future features will include using Entando document storage for
-storing documents and a pluggable document management interface.
+## Configuration
+ 
+The following discussion guides the configuration of PDA features and capabilities while introducing the plugin building blocks.
+ 
+For a discussion of components in the context of deployment, see [PDA Deployment Architecture](../../docs/concepts/pda-architecture.html#pda-mfes).
 
-## New Process Form
+## Application Details
 
-The new process form renders a form that allows the final user to
-provide the information necessary to instantiate a new business process
-instance. The technology for generating the process form is the same as
-the JSON schema definition used for task forms. === PAM Implementation
-The PAM implementation depends on a form definition being attached to
-the process definition. Entando transforms the PAM representation into a
-JSON schema form that can be rendered to the final user.
+The Entando Standard Banking Demo application demonstrates a number of the major features of the Entando platform, including
 
-## Summary Card
+- Micro Frontends (MFEs)
+- Process Automation Manager (PAM)
+- Backend for Frontend (BFF)
+### Micro Frontends (MFEs)
 
-The summary card MFE provides a view into aggregate data for the process
+Below is an overview of each micro frontend that is available as part of the Entando PDA plugin. Details specific to the PAM implementation of an MFE are provided where appropriate. MFE behavior and datasources must be defined if the integration layer is extended to other
+engines or custom implementations.
+#### Task List
```

Comment:
I hate to say it but a screenshot for each MFE might be really useful in the final write up. They do help show what you'll get out of the box for the Standard Demo.